src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java

```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////////
 // checkstyle: Checks Java source code for adherence to a set of rules.
-// Copyright (C) 2001-2014  Oliver Burn
+// Copyright (C) 2001-2020 the original author or authors.
```

Comment:
Can we remove content of this line at all, we do not need to update this file each year.

Reply:
@romani do you want me to remove header from all Input files that don't need it? There are more if you look at diff from my update.

https://github.com/checkstyle/checkstyle/commit/ae9edbd4a8645c8cc1a99ad6890dd0006700d1cb#diff-c04011f5de5dd3db31ac528406e0b465R3

Reply:
removed.